vendor/src/github.com/hashicorp/go-plugin/LICENSE

```diff
@@ -0,0 +1,353 @@
+Mozilla Public License, version 2.0
```

Comment:
Really? MPL is copyleft, whereas Apache is doesn't have such distribution restrictions. I don't see MPL anywhere on [here](https://www.docker.com/components-licenses). There is some GPLv3 but that specifically has a static linking exception.

IANAL, but this requires more thought. For example, this may have implications for proprietary authorization plugins.